server/graphql/general/src/queries/store.rs

```diff
@@ -141,6 +143,7 @@ impl StoreFilterInput {
             name: name.map(SimpleStringFilter::from),
             name_code: name_code.map(SimpleStringFilter::from),
             site_id: site_id.map(EqualFilter::from),
+            logo:None,
```

Comment:
```suggestion
            logo: None,
```
There is a feature in VSCode "format on save" which should work the Rust formatter. So if you safe your file it will to the proper formatting automatically...